src/include/add-on/add-on-workflow.rb

```diff
@@ -1897,6 +1897,32 @@ def media_type_selection
 
   private
 
+    #
+    # Offer the add-ons from the installation medium automatically?
+    # The addons are only offered when:
+    #   - Running in the first stage
```

Comment:
this is new part of code, it is to avoid calling medium type? Can it happen?

Reply:
Yes, that's to avoid querying the medium type in an installed system. I'm not sure, but this code could be also used in an installed system, so just to be sure...